packages/pluggableWidgets/image-native/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
The automation benefits from having an item below unreleased, such as "- introduced", when it builds the changelog/notes. I think we discovered this awhile back and spoke about a convention here.